components/shared/modals/BoxedModal.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div class="modal-width">
+    <header
+      class="px-6 is-flex is-justify-content-space-between border-bottom is-align-items-center"
+      :class="{
+        'py-5': !loading,
+        'py-4': loading,
+      }">
```

Comment:
that's a bit odd
can't we stick to one of them?

Reply:
base is py-5

loading needs to be 4 since the loader is bigger than the text otherwise there is a change of the box size here 

![CleanShot 2023-11-20 at 12 30 36](https://github.com/kodadot/nft-gallery/assets/44554284/dcaa529a-e2a1-4eb7-8ed0-7ec487269640)